examples/async_tcp_stream/src/main.rs

```diff
@@ -0,0 +1,33 @@
+use std::io;
+use tokio::io::{AsyncReadExt, AsyncWriteExt};
```

Comment:
remove tokio

Reply:
`AsyncReadExt` provides extra methods like `read_to_string`. And `AsyncWriteExt` provides `write_all`.
So if we remove them, we need to implement these traits.